website/src/pages/tutorial.md

```diff
@@ -0,0 +1,530 @@
+    <p>John Doe: Building and breaking things</p>
+    <p>Jane Smith: The development process</p>
+    <p>Matt Miller: The Web 7.0</p>
+    <p>Tom Jerry: Mouseless development</p>
+</div>
+<div>
+    <h3>John Doe: Building and breaking things</h3>
+    <img src="https://via.placeholder.com/640x360.png?text=Video+Player+Placeholder" alt="video thumbnail">
+</div>
+```
+
+Now, let's convert this HTML into `html!`. Type (or copy/paste) the following snippet into the body of `app` function
+such that the value of `html!` is returned by the function
+
+```rust
+html! {
+    <>
+        <h1>{ "RustConf Explorer" }</h1>
+        <div>
+            <h3>{"Videos to watch"}</h3>
+            <p>{ "John Doe: Building and breaking things" }</p>
+            <p>{ "Jane Smith: The development process" }</p>
+            <p>{ "Matt Miller: The Web 7.0" }</p>
+            <p>{ "Tom Jerry: Mouseless development" }</p>
+        </div>
+        <div>
+            <h3>{ "John Doe: Building and breaking things" }</h3>
+            <img src="https://via.placeholder.com/640x360.png?text=Video+Player+Placeholder" alt="video thumbnail" />
+        </div>
+    </>
+}
+```
+
+Refresh the browser page, and you should see the following output displayed:
+
+**TODO: Add screenshot**
+
+### Using Rust language constructs in the markup
+
+A big advantage of writing markup in Rust is that we get all the coolness of Rust in our markup.
+Now, instead of hardcoding the list of videos, let's actually define them as a `Vec` of Rust objects.
+We'll create a simple `struct` (in `main.rs` or any file of our choice) which will hold our data.
+
+```rust
+struct Video {
+    id: usize,
+    title: String,
+    speaker: String,
+    url: String,
+}
+```
+
+Next, we'll create instances of this struct in our `app` function and use those instead of hardcoding the data:
+
+```rust
+let videos = vec![
+    Video {
+        id: 1,
+        title: "Building and breaking things".to_string(),
+        speaker: "John Doe".to_string(),
+        url: "https://youtu.be/PsaFVLr8t4E".to_string(),
+    },
+    Video {
+        id: 2,
+        title: "Building and breaking things".to_string(),
+        speaker: "Jane Smith".to_string(),
+        url: "https://youtu.be/PsaFVLr8t4E".to_string(),
+    },
+    Video {
+        id: 3,
+        title: "The development process".to_string(),
+        speaker: "Matt Miller".to_string(),
+        url: "https://youtu.be/PsaFVLr8t4E".to_string(),
+    },
+    Video {
+        id: 4,
+        title: "The Web 7.0".to_string(),
+        speaker: "Tom Jerry".to_string(),
+        url: "https://youtu.be/PsaFVLr8t4E".to_string(),
+    },
+];
+```
+
+In order to display them, we need to convert these `Vec`s into `Html`. We can do that by creating an iterator, 
+mapping it to `html!` and collecting it as `Html`:
+
+```rust
+let videos = videos.iter().map(|video| html! {
+    <p>{format!("{}: {}", video.speaker, video.title)}</p>
+}).collect::<Html>();
+```
+
```

Comment:
The html here and the one in the previous section are not identical. This might confuse the reader. The code snippet here should either be fixed or it should be explicitly that this is only the video list html.

Reply:
I think it is fairly obvious that this is just the videos list as it's mapping the videos and doing nothing else.